ibis/expr/schema.py

```diff
@@ -153,6 +153,34 @@ def from_tuples(
         # construct the schema
         return cls(dict(zip(names, types)))
 
+    @classmethod
+    def from_numpy(self, numpy_schema):
+        """Return the equivalent ibis schema."""
+        from ibis.formats.numpy import schema_from_numpy
+
+        return schema_from_numpy(numpy_schema)
+
+    @classmethod
+    def from_pandas(self, pandas_schema):
+        """Return the equivalent ibis schema."""
+        from ibis.formats.pandas import schema_from_pandas
```

Comment:
Does this need some tests?

Reply:
`schema_from_pandas` is covered just the method isn't

Reply:
Covered it with a test.